components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -72,15 +74,20 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
   // Empty Page
   profilesEmpty = false;
 
+  public isAvailableProfilesVisible = false;
+
   private isDestroyed: Subject<boolean> = new Subject<boolean>();
 
   constructor(
     private profilesService: ProfilesService,
     private availableProfilesService: AvailableProfilesService,
     private uploadService: UploadService,
     private chefSessionService: ChefSessionService,
-    private layoutFacade: LayoutFacadeService
-  ) {}
+    private layoutFacade: LayoutFacadeService,
+    private productDeployedService: ProductDeployedService
+  ) {
+    this.isAvailableProfilesVisible = !this.productDeployedService.isProductDeployed('desktop');
+  }
```

Comment:
If the Desktop offering is installed the available profiles should not be visible.